packages/palette/package.json

```diff
@@ -43,6 +43,7 @@
   },
   "homepage": "https://github.com/artsy/palette#readme",
   "peerDependencies": {
+    "luxon": "^1.15.0",
```

Comment:
Let’s remove the last version component and allow any `1.x` version, starting from `1.15`.